dev-manuals/services/user-service.md

```diff
@@ -2,6 +2,8 @@
 This service is dedicated to handling all use cases related to user data.
 This service acts internally (in our system) as a graphQL endpoint extension for our [Keycloak application](../authentication/keycloak.md) that stores and manages all registered users in a system. This is necessary as Keycloak only supports RESTful request messages. The [gateway service](./gateway-service.md) queries user data over this endpoint.
 
+All information pertaining to course memberships and roles within a course are located here. Information about the user and their course memberships (including their roles within the course) can be requested from this service.
```

Comment:
this is wrong. It's located in the course service

Reply:
in review guide, this is in user service @PaulBredl

Reply:
The review guide is two weeks old. Feel free to check out the code if you don't believe me 😅

Reply:
no no, its fine, I just told like in review guide it is like that, but I will remove this